udfs/community/test_cases.js

```diff
@@ -1449,6 +1449,24 @@ generate_udf_test("cw_regexp_replace_4", [
     expected_output: `"TestCad$123456"`,
   },
 ]);
+generate_udf_test("cw_regexp_replace_4", [
+  {
+    inputs: [`"TestStr123456Str"`, `"Str"`, `"Cad$"`, `CAST(1 AS INT64)`],
+    expected_output: `"TestCad$123456Cad$"`,
+  },
+]);
+generate_udf_test("cw_regexp_replace_4", [
+  {
+    inputs: [`"abaa"`, `"a"`, `"A"`, `CAST(1 AS INT64)`],
+    expected_output: `"AbAA"`,
+  },
+]);
+generate_udf_test("cw_regexp_replace_4", [
+  {
+    inputs: [`"abaa"`, `"a"`, `"A"`, `CAST(2 AS INT64)`],
+    expected_output: `"abAA"`,
+  },
+]);
```

Comment:
This test is failing with the following error:
expected "abAA", but saw "bAA".

Reply:
Oh, thanks, sent a patch. Is there a way I can see the test failure logs myself? Is it in one of those 45 checks?

Reply:
Yes it's one of those 45 checks (the one named "bigquery-utils-push-to-pr"). Here's the check that was failing: https://github.com/GoogleCloudPlatform/bigquery-utils/pull/450/checks?check_run_id=30891233416